pkg/rancher-desktop/assets/specs/command-api.yaml

```diff
@@ -358,6 +358,10 @@ components:
                         cacheMode:
                           type: string
                           enum: ['none', 'loose', 'fscache', 'mmap']
+            rdNetworking:
```

Comment:
I realize this is just an "experimental" name, but `rd-networking` looks kind of odd to me. All settings are about Rancher Desktop, so prefixing it with `rd` doesn't make sense to me. I wonder if we can't come up with a more descriptive name?

Reply:
The name is derived from the repo/project `rancher-desktop-netwoking`. It looks like before we were using the following scheme for our experimental settings `experimental+project` to indicate that this is an experimental flag e.g `experimentalHostResolver` but, now that we have an experimental parent object I thought it makes sense just to use the project name.

Reply:
When I talked about this with @mook-as we agreed to use the name the setting would have once/if it graduates to be non-experimental, and then prefix it with experimental:

```
$ rdctl set --help
Update selected fields in the Rancher Desktop UI and restart the backend.

Usage:
  rdctl set [flags]

Flags:
      --application.admin-access
      --application.auto-start
      --application.debug
      --application.hide-notification-icon
      --application.path-management-strategy string                     (Allowed values: [manual, rcfiles])
      --application.start-in-background
      --application.telemetry.enabled
      --application.updater.enabled
      --application.window.quit-on-close
      --container-engine.allowed-images.enabled
      --container-engine.allowed-images.locked
      --container-engine.name string                                    Set engine to containerd or moby (aka docker). (Allowed values: [containerd, docker, moby])
      --diagnostics.show-muted
      --experimental.virtual-machine.mount.9p.cache-mode string         (Allowed values: [none, loose, fscache, mmap])
      --experimental.virtual-machine.mount.9p.msize-in-kb int
      --experimental.virtual-machine.mount.9p.protocol-version string   (Allowed values: [9p2000, 9p2000.u, 9p2000.L])
      --experimental.virtual-machine.mount.9p.security-model string     (Allowed values: [passthrough, mapped-xattr, mapped-file, none])
      --experimental.virtual-machine.mount.type string                  (Allowed values: [reverse-sshfs, 9p, virtiofs])
      --experimental.virtual-machine.socket-vmnet
      --experimental.virtual-machine.type string                        (Allowed values: [qemu, vz])
      --experimental.virtual-machine.use-rosetta
  -h, --help                                                            help for set
      --images.namespace string
      --images.show-all
      --kubernetes.enabled                                              Control whether kubernetes runs in the backend.
      --kubernetes.options.flannel                                      Control whether flannel is enabled. Use to disable flannel so you can install your own CNI.
      --kubernetes.options.traefik
      --kubernetes.port int
      --kubernetes.version string                                       Choose which version of kubernetes to run.
      --port-forwarding.include-kubernetes-services
      --virtual-machine.memory-in-gb int
      --virtual-machine.number-cpus int
```

The top level "bucket" should correspond to the Preferences dialog page where we would put the UI for changing the setting.

Reply:
FWIW, once the `rancher-desktop-networking` graduates we would not have any settings for it. It would just be the default network stack for RD and there will be no alternatives.